gazebo/msgs/performance_metrics.proto

```diff
@@ -0,0 +1,21 @@
+syntax = "proto2";
+package gazebo.msgs;
+
+/// \ingroup gazebo_msgs
+/// \interface PerformanceMetrics
+/// \brief A message for run-time simulation performance metrics
+
+message PerformanceMetrics
+{
+
+  message PerformanceSensorMetrics
+  {
+    required string sensor_name             = 1;
+    required double real_sensor_update_rate = 2;
+    required double sim_sensor_update_rate  = 3;
```

Comment:
depends on your style, but "sensor" seems redundant given the outer message name. And if you keep "Sensor", should you add it to the "fps" field below?